abstractions/php/composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "microsoft/kiota-abstractions",
```

Comment:
if you have a repository URL field, you might want to set it to https://github.com/microsoft/kiota for the publishing phase.
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#publishing-a-package-using-a-nugetconfig-file